config/config.exs

```diff
@@ -7,18 +7,18 @@
 # General application configuration
 import Config
 
-config :app,
-  ecto_repos: [App.Repo]
+config :spike_papertrail,
```

Comment:
@LuchoTurtle please avoid wasting your time with this sort of renaming.
We use `App` as our standard name for examples for a _very_ good reason; to avoid this time-suck.
The GitHub repository has a descriptive name. 
The `Phoenix` App doesn't need one.

Reply:
I did this so that if people went from the `todo-tutorial` to this one, they wouldn't have trouble with database names.
Do you want me to rename it back to `App` or is it another "time-suck"?